yb-voyager/src/query/queryissue/parser_issue_detector.go

```diff
@@ -798,12 +821,172 @@ func (p *ParserIssueDetector) IsUnloggedTablesIssueFiltered() bool {
 	return p.isUnloggedTablesIssueFiltered
 }
 
+// GetForeignKeyConstraints returns all foreign key constraints
+func (p *ParserIssueDetector) GetForeignKeyConstraints() []ForeignKeyConstraint {
+	return p.foreignKeyConstraints
+}
+
+// GetTableIndexes returns the table indexes map
```

Comment:
Can't find where these getters are being used. let's remove them if they are not being used

Reply:
Yep residue from older implemetation. Removed.